proc/threads.c

```diff
@@ -432,14 +431,7 @@ static void _threads_updateWakeup(time_t now, thread_t *min)
 
 static inline time_t _threads_getTimer(void)
```

Comment:
After your changes, `_threads_getTimer` invokes only `hal_getTimer` and doesn't do anything else. 
What do you think about removing `_threads_getTimer` and using only `hal_getTimer`?

Reply:
I was planning on doing that in the next commits, but can change it now as well